pandas/tests/test_categorical.py

```diff
@@ -556,28 +556,35 @@ def test_categories_none(self):
     def test_describe(self):
         # string type
         desc = self.factor.describe()
+        self.assertTrue(self.factor.ordered)
+        exp_index = pd.CategoricalIndex(['a', 'b', 'c'], ordered=True,
+                                        name='categories')
         expected = DataFrame({'counts': [3, 2, 3],
                               'freqs': [3 / 8., 2 / 8., 3 / 8.]},
-                             index=pd.CategoricalIndex(['a', 'b', 'c'],
-                                                       name='categories'))
```

Comment:
are we explicity forcing this to ordered? (not against it, more curious where/why we did this)